Source/WebCore/editing/EditingStyle.cpp

```diff
@@ -536,12 +536,9 @@ void EditingStyle::init(Node* node, PropertiesToInclude propertiesToInclude)
 
 void EditingStyle::removeTextFillAndStrokeColorsIfNeeded(const RenderStyle* renderStyle)
 {
-    // If a node's text fill color is invalid, then its children use 
-    // their font-color as their text fill color (they don't
-    // inherit it).  Likewise for stroke color.
-    if (!renderStyle->textFillColor().isValid())
+    if (RenderStyle::isCurrentColor(renderStyle->textFillColor()))
         m_mutableStyle->removeProperty(CSSPropertyWebkitTextFillColor);
-    if (!renderStyle->textStrokeColor().isValid())
+    if (RenderStyle::isCurrentColor(renderStyle->textStrokeColor()))
```

Comment:
This can just test renderStyle->textStrokeColor().isCurrentColor().
RenderStyle::isCurrentColor can be removed (now or later).

Reply:
Yes `RenderStyle::isCurrentColor` will be removed in a future patch.